corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -696,6 +708,9 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
                             return 'Lookup table tag required';
                         }
                     }
+                    if (!self.case_type()) {
+                        return 'Expected case type required';
```

Comment:
This validation doesn't actually prevent the user from saving (strangely) so it shouldn't have any effect in the example you mention.